packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/list.html

```diff
@@ -97,6 +100,7 @@
         data-sortable
         data-filterable
     >
+        <span data-ng-bind="$value | bytes:2:true:'GiB'"></span>
```

Comment:
**nitpick:** I think `GiB` needs translation

Reply:
No, `GiB` indicate the base unit for storage capacity.
Here we indicate that capacity is GiB main unit.
Sample: `capacity: 10` means `10GiB`, if `capacity:1000` it will display `1TiB` and so on